src/chains/tests/test_models.py

```diff
@@ -159,6 +159,41 @@ def test_valid_addresses(self) -> None:
                 chain.full_clean()
 
 
+class ChainTransactionServiceUrlValidationTestCase(TestCase):
+    def test_invalid_urls(self) -> None:
+        param_list = [
+            "foo://bar",
+            "foo",
+            "://",
+        ]
+        for invalid_url in param_list:
+            with self.subTest(msg=f"{invalid_url} is not a valid url"):
+                with self.assertRaises(ValidationError):
+                    chain = ChainFactory.create(transaction_service_uri=invalid_url)
+                    chain.full_clean()
+
+            with self.subTest(msg=f"{invalid_url} is not a valid url"):
+                with self.assertRaises(ValidationError):
+                    chain = ChainFactory.create(vpc_transaction_service_uri=invalid_url)
+                    chain.full_clean()
+
+    def test_valid_urls(self) -> None:
+        param_list = [
+            "http://tx-service",
+            "https://tx-service",
```

Comment:
Maybe you can add one with port `https://tx-service:8000`

Reply:
Done in [d381e3b](https://github.com/gnosis/safe-config-service/pull/468/commits/d381e3ba966f02b54f8766ca862673c356f8cc20)